model_zoo/census_model_sqlflow/wide_and_deep/transform_ops.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+
+
+class TransformOpType(Enum):
```

Comment:
Seems transform ops will also be used in SQLFlow in the future, so can `TransformOp` be an independent python package so that we can install it by `pip install` or will this be replaced by Keras transform layers?

Reply:
Yes, the `TransformOp` will be in the SQLFlow code. It's used in the code_generator component.
I just want to describe the data structure and choose to write here.